jenkins/Jenkinsfile

```
@@ -55,7 +79,52 @@ pipeline {
             '''
           }
         }
+        stage('NDTensorscuTENSORExt julia-1.11') {
+	        options {
+            timeout(time: 45, unit: 'MINUTES')
+          }
+          agent {
+            dockerfile {
+              label 'gpu&&v100'
+              filename 'Dockerfile'
+              dir 'jenkins'
+              additionalBuildArgs  '--build-arg JULIA=1.11'
+              args '--gpus "device=1"'
+            }
+          }
+          environment {
+            HOME = pwd(tmp:true)
+            OMP_NUM_THREADS = 4
+            JULIA_NUM_THREADS = 4
+          }
+          steps {
+            sh '''
+              julia -e 'using Pkg; Pkg.activate(temp=true); Pkg.Registry.update(); Pkg.update(); Pkg.develop(path="./NDTensors"); Pkg.develop(path="."); Pkg.test("NDTensors"; test_args=["cutensor"])'
+            '''
+          }
+        }
         stage('NDTensorsMetalExt Julia-1.10'){
+          options {
+            timeout(time: 45, unit: 'MINUTES')
+          }
+          agent {
+            label 'm1'
+          }
+          environment{
+            PATH="${env.HOME}/.juliaup/bin:${env.PATH}"
+            PLATFORM = 'macos'
+          }
+          steps{
+            sh '''
+              juliaup add 1.10
+              juliaup default 1.10
+            '''
+            sh '''
+              julia -e 'using Pkg; Pkg.activate(temp=true); Pkg.develop(path="./NDTensors"); Pkg.develop(path="."); Pkg.test("NDTensors"; test_args=["metal"])'
+            '''
+          }
+        }
+        stage('NDTensorsMetalExt Julia-1.11'){
```

```suggestion
        stage('NDTensorsMetalExt Julia-1'){
```
Same reason as above.

---

updated